server/db/index.js

```diff
@@ -1,8 +1,10 @@
 var mysql      = require('mysql');
+require('dotenv').config();
+
 var connection = mysql.createConnection({
-  host     : 'localhost',
-  user     : 'root',
-  database : 'twitchy',
+  host     : process.env.DB_HOST,
+  user     : process.env.DB_USER,
+  database : process.env.DB_DATABASE,
```

Comment:
Suggest also updating your readme to talk about this. You could also update the code to display a useful error if they are left blank.

Reply:
I referred to your README.md with the display error are you saying out it in the readme of in this very file?

Reply:
I think he means in your readme, include instructions to set up the .env file.

Reply:
Sorry for the confusion. This pull makes use of process.env.DB_HOST for instance. So I was suggesting that you also change readme.md to say something like "Before you run this service, set the following environment variables: DB_HOST..."